charts/opentelemetry-collector/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v2
 name: opentelemetry-collector
-version: 0.104.0
+version: 0.105.2
```

Comment:
This will now need to be `0.106.2`  since the version has already been bumped up: https://github.com/open-telemetry/opentelemetry-helm-charts/blob/main/charts/opentelemetry-collector/Chart.yaml#L3

Also, please run `make generate-examples CHARTS=opentelemetry-collector` to update the examples' versions.